src/js/pannellum.js

```diff
@@ -1412,7 +1438,8 @@ function animate() {
     } else if (renderer && (renderer.isLoading() || (config.dynamic === true && update))) {
         requestAnimationFrame(animate);
     } else {
-        fireEvent('animatefinished', {pitch: _this.getPitch(), yaw: _this.getYaw(), hfov: _this.getHfov()});
+        if (_this.getPitch && _this.getYaw && _this.getHfov)
```

Comment:
Why is this check necessary?

Reply:
Yes, with the following steps I can reproduce an error:
1. Use the source files of the viewer
2. Instantiate a viewer (from javascript)
   * Pass a first scene
   * Set autoLoad = true of the scene
   * The scene type is equirectangular
3. The call stack is the following
   viewer -> Viewer -> processOptions -> init -> onImageLoad -> renderInit -> init  -> renderInitCallback -> animateInit -> animate 
4. Line 1438 will evaluate to false (due to type equirectangular) but `_this` has not yet the functions getPitch() ... registered.
5. Line 1442 fails to execute

Reply:
Okay, that makes sense. It's a configuration I've never tested.